cmd/utils.go

```diff
@@ -95,9 +96,92 @@ func splitStr(path, sep string, n int) []string {
 	return splits
 }
 
+// Parse scheme format, supported formats are
+// - "http"
+// - "http+s3v2"
+// - "http+s3v4"
+func parseScheme(scheme string) (string, string, *probe.Error) {
```

Comment:
Can you lower string scheme ? so scheme can be case insensitive

Reply:
Good idea will add a unit test.